docs/other-tools/nfs/nfs-server-client-setup.md

```diff
@@ -0,0 +1,259 @@
+# Network File System (NFS)
+
+NFS enables a system to share directories and files with others over a network.
+With NFS, users and applications can access files on remote systems as if they
+were stored locally. Client systems mount a directory hosted on the NFS server,
+allowing them to access and work with the files it contains.
+
+## Pre-requisite
+
+We are using following setting for this purpose to setup the NFS server and
+client on Ubuntu based NERC OpenStack VM:
+
+-   1 Linux machine for the **NFS Server**, `ubuntu-22.04-x86_64`, `cpu-su.1` flavor
```

Comment:
Latest Ubuntu LTS relase is 24.04 (https://ubuntu.com/about/release-cycle)  I wonder if it would be worth specifying that, instead of 22.04, albeit 22.04 is full production until Apr 2027.

Reply:
I can revise with that version soon. Thanks @aabaris

Reply:
All tested with `ubuntu-24.04-x86_64` and verified same scripts works fine.